### PR TITLE
Include max-params

### PR DESCRIPTION
**Pull request by thalesmello**
Because of the lack of a consensus on #12, at least we should make this problem less worse by limiting the number of max parameters.

### CODE REVIEW

**Comment by thalesmello** [comment]
@mccraveiro I reworded the commit. Could you take a look please?

By the way, I just noticed the majority of developers don't capitalize the subject, as the style-guide mandates. Please refer to https://github.com/pagarme/git-style-guide/pull/8 .

File: .eslintrc.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ module.exports = {
   rules: {
     semi: [2, 'never'],
     'space-before-function-paren': [2, 'always'],
+    'max-params': ["error", { max: 3 }],
```
Comment (mccraveiro) [inline .eslintrc.js]: Please use single quotes on `error`

**Comment by thalesmello** [comment]
@mccraveiro Good catch. Didn't notice that one.

**Comment by thalesmello** [comment]
Let's vote about this rule. Thumps up who is for, thumbs down who is against.

**Pull request merged**